__tests__/news.test.js

```diff
@@ -258,4 +258,31 @@ describe('/api/articles/:article_id/comments', () => {
             });
     })
 })
+});
+
+describe('/api/users', () => {
+    describe('GET users', () => {
+        test('returns an array which is greater than length 0', () => {
+            return request(app).get('/api/users')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.length).not.toBe(0);
+            });
+        });
+        test('returns all users as an array of objects with the appropriate properties', () => {
+            return request(app).get('/api/users')
+            .expect(200)
+            .then(({body}) => {
```

Comment:
Always check the length of an array thats coming back before you iterate through it

Reply:
The test above checked the length of the array - is this meant to be in the same test? Thanks! :)